js assignment4/Registration_Form/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Registration Form</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+  <link rel="stylesheet" href="assets/css/style.css">
+</head>
+<body>
+<div class="">
+<div class="container">
+  <div class="d-flex justify-content-center">
+  <h2>Registration Form</h2>
+  </div>
+  <form name="Form" method="get" action="/" onsubmit="validate(event)">
+    <div class="form-group">
+        <label for="Name">First Name:</label>
+        <input type="text" class="form-control" id="Name" placeholder="Enter Name" name="Name">
+        <span id="name_validate"></span>
+    </div>
+    <div class="form-group">
+        <label for="Last-Name">Last Name:</label>
+        <input type="text" class="form-control" id="Last-Name" placeholder="Enter Last Name" name="LastName">
+        <span id="lname_validate"></span>
+    </div>
+    <div class="form-group">
+        <label for="Age">Age:</label>
+        <input type="text" class="form-control" id="Age" placeholder="Enter Age" name="Age">
+        <span id="Age_validate"></span>
+    </div>
+    <div class="form-group">
+      <label for="email">Email:</label>
+      <input type="text" class="form-control" id="email" placeholder="Enter email" name="email">
+      <span id="validate"></span>
+    </div>
+    <div class="form-group">
+      <label for="Phone">Phone Number:</label>
+      <input type="text" class="form-control" id="Phone" placeholder="Enter Phone Number" name="Phone">
+      <span id="Phone_validate"></span>
+    </div>                
+    <div class="form-group">
+        <label for="Address">Address:</label>
+        <input type="text" class="form-control" id="Address" placeholder="Enter Address" name="Address">
+        <span id="Address_validate"></span>
+    </div>
+    
+    <input type="submit" class="btn btn-outline-primary"    value="Submit">
+    <button type="button" class="btn btn-outline-danger" onclick="removee(event)">Reset</button>

```

Comment:
Better practice to use addEventListener instead of inline event listener
Read: event bubbling in js